conda/install.py

```diff
@@ -922,6 +927,8 @@ def move_path_to_trash(path):
         except OSError as e:
             log.debug("Could not move %s to %s (%s)" % (path, trash_dir, e))
         else:
+            if path in linked_data_:
+                del linked_data_[path]
```

Comment:
Note that `linked_data_` contains one entry per *prefix*. Can't `path` be anything? If so, this is not going to clean out the `linked_data_` properly. The safest thing to do is wipe out the entire prefix. The efficient thing to do is to know the full name (including channel prefix) of the package being removed, and `del linked_data_[prefix][dist]`.